woocommerce-services.php

```diff
@@ -788,8 +788,16 @@ private function dismiss_tos_notice() {
 		 * @return bool
 		 */
 		private function can_accept_tos() {
+			if ( defined( 'JETPACK_DEV_DEBUG' ) && JETPACK_DEV_DEBUG && current_user_can( 'manage_options' ) ) {
+				return true;
+			}
+
+			if ( ! class_exists( 'Jetpack_Data' ) ) {
+				return false;
+			}
+
 			$user_token = Jetpack_Data::get_access_token( JETPACK_MASTER_USER );
-			return get_current_user_id() === $user_token->external_user_id;
+			return is_object( $user_token ) && isset( $user_token->external_user_id) && get_current_user_id() === $user_token->external_user_id;
```

Comment:
You should be able to skip the `is_object()` check here and shorten this line a little. Also, missing a space in the `isset()` check.